src/Microsoft.Framework.Runtime.Abstractions/ILibraryManager.cs

```diff
@@ -11,24 +11,24 @@ namespace Microsoft.Framework.Runtime
     /// </summary>
     public interface ILibraryManager
```

Comment:
I think it's worth breaking this type apart since you're doing all of these other changes. Basically anything with LibraryExport move to another type in Microsoft.Framework.Compilation.Abstractions